backend/templates/auth/login.html

```diff
@@ -0,0 +1,11 @@
+<form method="POST">
```

Comment:
This works, but it's way too basic. You're not extending your base.html template. Could you please try extending from it and making this login form have at least your site's basic look-and-feel?

Reply:
Yeah, I'm going to fix it. I was having a hard time styling the form's text because it wasn't changing from white to black. I was gonna wait until the meeting later to get some help